sw/device/silicon_creator/imm_rom_ext/defs.bzl

```diff
@@ -12,31 +12,19 @@ DEFAULT_EXEC_ENV = [
 
 # The target list should contian prebuilt artifacts and run-time build targets.
 SLOT_A_IMM_ROM_EXT_SECTIONS = {
-    "main": [
-        "//sw/device/silicon_creator/imm_rom_ext:main_section_slot_a_fpga_cw310",
-        "//sw/device/silicon_creator/imm_rom_ext:main_section_slot_a_fpga_cw340",
-        "//sw/device/silicon_creator/imm_rom_ext:main_section_slot_a_sim_dv_base",
-        "//sw/device/silicon_creator/imm_rom_ext:main_section_slot_a_sim_verilator_base",
-        "//sw/device/silicon_creator/imm_rom_ext:main_section_slot_a_silicon_creator",
-    ],
+    "main": "//sw/device/silicon_creator/imm_rom_ext:main_section_slot_a",
```

Comment:
Are these `SLOT_*_IMM_ROM_EXT_SECTIONS` consts even needed any more? It looks like the targets within are just used in other target's `deps`. Can you for example just use the dep directly, as shown below?